### PR TITLE
Restore docker.network.in.* and docker.network.out.* fields in docker module

### PR DESCRIPTION
**Pull request by MichaelKatsoulis**
## Proposed commit message

- WHAT: Restore docker.network.in.* and docker.network.out.* fields in docker module
- WHY:  These fields contain network incoming and outgoing stats per seconds and should not have been deprecated
in favour of `docker.network.inbound` and `docker.network.outbound` which contain network stats since the container started.

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## How to test this PR locally

1.  Package metricbeat for docker `PLATFORMS=linux/arm64 PACKAGES=docker  mage package`
2. Bring up elastic-stack `elastic-package stack up -d -v --version 9.0.0-SNAPSHOT`
3. In a machine (VM or local) with docker installed run Metricbeat by:
```
docker run -d \                                                                             
  --name=metricbeat \ 
  --user=root \
  --volume="$(pwd)/metricbeat.docker.yml:/usr/share/metricbeat/metricbeat.yml:ro" \
  --volume="/var/run/docker.sock:/var/run/docker.sock:ro" \
  --volume="/sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro" \
  --volume="/proc:/hostfs/proc:ro" \
  --volume="/:/hostfs:ro" \
  docker.elastic.co/beats/metricbeat-oss:9.0.0 metricbeat -e \
  -E output.elasticsearch.hosts=["https://elasticsearch:9200"] -E output.elasticsearch.ssl.verification_mode=none -E  output.elasticsearch.username=elastic -E output.elasticsearch.password=changeme
```
4. Explore the docker.network.in.* and docker.network.out.*  in Kibana discover

## Related issues

-

## Screenshots

docker network

### CODE REVIEW

**Comment by pierrehilbert** [comment]
@MichaelKatsoulis it seems strange to me to see the CLA being unhappy with your contribution.

**Comment by gizas** [comment]
@MichaelKatsoulis we will need an integration upgrade [right](https://github.com/elastic/integrations/blob/main/packages/docker/data_stream/network/fields/fields.yml)? As I dont see the fields there

**Comment by MichaelKatsoulis** [comment]
> @MichaelKatsoulis it seems strange to me to see the CLA being unhappy with your contribution.

Maybe it is because I  just got new laptop and I wrongly configured the email for git

**Comment by MichaelKatsoulis** [comment]
> @MichaelKatsoulis we will need an integration upgrade [right](https://github.com/elastic/integrations/blob/main/packages/docker/data_stream/network/fields/fields.yml)? As I dont see the fields there

Yes the package will be updated accordingly whet it is time.

File: metricbeat/module/docker/network/_meta/fields.yml [hunk 1]
```diff
@@ -9,6 +9,50 @@
...
+      fields:
+        - name: bytes
+          type: long
+          format: bytes
+          description: >
+            Incoming bytes per seconds.
+        - name: dropped
+          type: scaled_float
+          description: >
+            Dropped incoming packets per second..
```
Comment (rdner) [inline metricbeat/module/docker/network/_meta/fields.yml:25]: Is the `..` at the end intended?

Comment (rdner) [reply metricbeat/module/docker/network/_meta/fields.yml:25]: It's in a few places down the file.

Comment (MichaelKatsoulis) [reply metricbeat/module/docker/network/_meta/fields.yml:25]: No. Thanks for noticing!

**Pull request merged**